src/Driver/SqsDriver.php

```diff
@@ -101,8 +102,10 @@ private function queueExists($queueName)
             return false;
         } catch (SqsException $exception) {
             if ($previousException = $exception->getPrevious()) {
-                if ($previousException->getCode() === self::AWS_SQS_EXCEPTION_BAD_REQUEST) {
-                    return false;
+                switch ($previousException->getCode()) {
+                    case self::AWS_SQS_EXCEPTION_BAD_REQUEST:
+                    case self::AWS_SQS_EXCEPTION_NOT_FOUND:
+                        return false;
```

Comment:
I would change this to a `if (in_array($previousException->getCode(), [self::AWS_SQS_EXCEPTION_BAD_REQUEST, AWS_SQS_EXCEPTION_NOT_FOUND]) {`. Makes it a bit shorter

Reply:
Intention was more toward future-proofing, in case we end up having to add any other codes.

Reply:
ok, good enough for me!